### ISSUE TITLE
Cross Validation 

### ISSUE THREAD

**Issue by Jahnvi13** [OpenGenus/cosmos#6564]
**This is a(n):**

- [X] New algorithm
- [ ] Update to an existing algorithm
- [ ] Error
- [ ] Proposal to the Repository

**Details:**

Cross-validation is a statistical method used to estimate the performance (or accuracy) of machine learning models. It is used to protect against overfitting in a predictive model, especially when the data is limited. I plan to add two (or more) types of cross validation techniques:
1. K fold Cross validation
2. Stratified K fold cross validation using sklearn library and write a tutorial on them

**Comment by Jahnvi13** [comment]
@AdiChat kindly approve and assign.
